packages/serialize/src/index.js

```diff
@@ -239,6 +239,9 @@ function createStringFromObject(
   } else {
     for (let key in obj) {
       let value = obj[key]
+      if (typeof value === 'function') {
+        value = value(mergedProps)
+      }
```

Comment:
I'm OK with this change. It's minimal and improves compatibility between different styling engines and compatibility. 

I think that we should only do this when `mergedProps !== undefined` and raise a warning otherwise, like we do here:
https://github.com/emotion-js/emotion/blob/cde417556b1d7fa5dc0f1774da9fb7dfca9caa32/packages/serialize/src/index.js#L174-L188

cc @emmatown , do you have any other thoughts about this one?

Reply:
Make sense. Added a check `mergedProps !== undefined` before calling the callback.